build/Dockerfile

```
@@ -121,58 +121,85 @@ CMD /bin/bash
 # STAGE: devnet-base
 #
 # This stage installs Foundry.
-FROM ${BASE_IMAGE} as devnet-base
+FROM golang:${GO_VERSION}-bookworm as devnet-base
 
 # Install system dependencies.
-ARG DEBIAN_FRONTEND=noninteractive
 RUN <<EOF
```

To keep consistency in this Dockerfile, use this `ARG` before each RUN that has an `apt` command.

```suggestion
ARG DEBIAN_FRONTEND=noninteractive
RUN <<EOF
```

---

Thanks for  the suggestion but, as we talked off line, I'll make sure this setting is uniform throughout the file.

That said, I'll resolve the other comments to keep this suggestion in a single thread.

---

Please take a look if the changes made at a18f82f6 suffice.